components/infobox/wikis/formula1/infobox_company_custom.lua

```diff
@@ -0,0 +1,66 @@
+---
+-- @Liquipedia
+-- wiki=formula1
+-- page=Module:Infobox/Company/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector', {requireDevIfEnabled = true})
+local Company = Lua.import('Module:Infobox/Company', {requireDevIfEnabled = true})
+
+local CustomCompany = Class.new()
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+local Title = Widgets.Title
+
+local CustomTeam = Class.new()
```

Comment:
```suggestion
```
overlooked that in the cleanup
makes no sense here and isn't used (hence the linter mentions it too)